apps/preview/next/pages/showcases/Alert/AlertPositive.tsx

```diff
@@ -0,0 +1,25 @@
+import { ShowcasePageLayout } from '../../showcases';
+// #region source
+import { SfIconCheckCircle, SfIconClose } from '@storefront-ui/react';
+
+export default function AlertPositive() {
+  return (
+    <div
+      role="alert"
+      className="flex items-start md:items-center bg-positive-100 pr-2 pl-4 ring-1 ring-positive-200 typography-text-sm md:typography-text-base py-1 rounded-md"
+    >
+      <SfIconCheckCircle className="mr-2 my-2 text-positive-700" />
+      <p className="py-2 mr-2">The product has been added to the cart.</p>
+      <button
+        type="button"
+        className="p-1.5 md:p-2 ml-auto rounded-md text-positive-700 hover:bg-positive-200 active:bg-positive-300 hover:text-positive-800 active:text-positive-900"
+        aria-label="Close alert"
```

Comment:
suggestion: if we have all those alerts on the same page, then maybe we can differentiate the aria labels, so its not just "close alert" 4 times? 😄 fe. 'Close positive alert' etc.

Reply:
https://github.com/vuestorefront/storefront-ui/pull/2590/commits/483678a16251c1a4e9ea97164ef1dc6a142aee78